docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -51,7 +51,7 @@ Load columns as:
 - Individual columns like `Size` and `CurrentPrices` in the `HousingData` class.
 - Multiple columns at a time in the form of a vector like `HistoricalPrices` in the `HousingData` class.
 
-If you have a vector property, apply the [`VectorType`](xref:Microsoft.ML.Data.VectorTypeAttribute) attribute to the property in your data model. It's important to note that all of the elements in the vector need to be the same type.
+If you have a vector property, apply the [`VectorType`](xref:Microsoft.ML.Data.VectorTypeAttribute) attribute to the property in your data model. It's important to note that all of the elements in the vector need to be the same type. Although loading columns individually provides flexibility when performing feature engineering, in data sets with a large number of columns, loading multiple columns as a single vector reduces the number of passes over the data when you apply data transformations.
```

Comment:
I know that "reduces the number of passes over the data" were the words that were used in the meeting this morning, but isn't is more that you have one float array to operate on rather than say 100 float arrays, and that is where the efficiency comes from. Perhaps we could just say it is more efficient.

Reply:
This expression might be a little clearer:

"We may want to say, "Keeping the columns separated allows for ease and flexibility of feature engineering, but for a very large number of columns (>N), operating on the individual columns causes a speed impact." (from the original issues)

Reply:
#resolved